internal/server/v1/variables/golden/bulk_variables/california.json

```diff
@@ -4825,41 +4825,11 @@
         "Count_LakeEffectSnowEvent",
         "Count_LightningEvent",
         "Count_MarineHighWindEvent",
-        "Count_MedicalConditionIncident_0To4Years_SummerSeason_ConditionHeatStress_PatientHospitalized",
-        "Count_MedicalConditionIncident_0To4Years_SummerSeason_ConditionHeatStress_VisitedEmergencyDepartment",
-        "Count_MedicalConditionIncident_0To4Years_SummerSeason_Female_ConditionHeatStress_PatientHospitalized",
-        "Count_MedicalConditionIncident_0To4Years_SummerSeason_Female_ConditionHeatStress_VisitedEmergencyDepartment",
-        "Count_MedicalConditionIncident_0To4Years_SummerSeason_Male_ConditionHeatStress_PatientHospitalized",
-        "Count_MedicalConditionIncident_0To4Years_SummerSeason_Male_ConditionHeatStress_VisitedEmergencyDepartment",
-        "Count_MedicalConditionIncident_15To34Years_SummerSeason_ConditionHeatStress_PatientHospitalized",
-        "Count_MedicalConditionIncident_15To34Years_SummerSeason_ConditionHeatStress_VisitedEmergencyDepartment",
-        "Count_MedicalConditionIncident_15To34Years_SummerSeason_Female_ConditionHeatStress_PatientHospitalized",
-        "Count_MedicalConditionIncident_15To34Years_SummerSeason_Female_ConditionHeatStress_VisitedEmergencyDepartment",
-        "Count_MedicalConditionIncident_15To34Years_SummerSeason_Male_ConditionHeatStress_PatientHospitalized",
-        "Count_MedicalConditionIncident_15To34Years_SummerSeason_Male_ConditionHeatStress_VisitedEmergencyDepartment",
-        "Count_MedicalConditionIncident_35To64Years_SummerSeason_ConditionHeatStress_PatientHospitalized",
```

Comment:
@manishvats2 @SudhishaK 
Can you please check if the SV deletions for EPHHeatRelatedIllness is expected?